website/app/components/doc/page/sidebar.hbs

```diff
@@ -1,9 +1,23 @@
-{{#if this.structuredPageTree}}
-  <aside class="doc-page-sidebar" ...attributes>
-    <div class="doc-page-sidebar__inner-wrapper">
+<aside class={{this.classNames}} ...attributes>
+  <div class="doc-page-sidebar__inner-wrapper">
+    <nav class="doc-page-sidebar__top-routes" aria-label="primary page navigation">
+      <ul>
```

Comment:
```suggestion
      <ul role="list">
```

If `list-style` is set to `none`, then the `<ul>` element needs the explicit role of `list` in order for Safari to relay the information correctly to assistive tech.